backend/mock/propelauth_fastapi_mock.py

```
@@ -0,0 +1,596 @@
+            send_email_to_confirm_email_address,
+            ask_user_to_update_password_on_login,
+            password,
+            username,
+            first_name,
+            last_name,
+            properties,
+            ignore_domain_restrictions,
+        )
+
+    def invite_user_to_org(
+        self, email: str, org_id: str, role: str, additional_roles: list[str] = []
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)